tests/openedx_tagging/core/tagging/test_api.py

```diff
@@ -75,13 +77,15 @@ def test_get_taxonomies(self):
 
         with self.assertNumQueries(1):
             both = list(tagging_api.get_taxonomies(enabled=None))
+        print(both)
```

Comment:
`print` left in ^

Reply:
Updated: 66b17c9a621f2f9cc0e72e087f4c6cc03696d365